plugins/catalog/api-report.md

```diff
@@ -351,7 +351,7 @@ export const EntityOrphanWarning: () => JSX.Element;
 
 // Warning: (ae-missing-release-tag) "EntityPageLayout" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
-// @public (undocumented)
+// @public @deprecated
```

Comment:
Perhaps add a note about this in the changeset?

Reply:
Totally, adding